docs/userguide/monitoring.rst

```diff
@@ -23,11 +23,20 @@ Here's an example configuration that logs monitoring information to a local sqli
 
 .. code-block:: python
 
+    import parsl
+    from parsl.monitoring.monitoring import MonitoringHub
+    from parsl.config import Config
+    from parsl.executors import HighThroughputExecutor
+    from parsl.addresses import address_by_hostname
+
+    import logging
+
     config = Config(
         executors=[
             HighThroughputExecutor(
                 label="local_htex",
                 cores_per_worker=1,
+                max_workers=4,
```

Comment:
Aren't we missing the monitoring config here ? Say include something like : 
```python
threads_config = Config(
    executors=[
	HighThroughputExecutor(
            label="local_htex",
            cores_per_worker=1,
            address=address_by_hostname(),
        )
    ],
    monitoring=MonitoringHub(
        hub_address=address_by_hostname(),
        hub_port=55055,
        logging_level=logging.INFO,
        resource_monitoring_interval=10,
    ),
    strategy=None
)
```

Reply:
It is already there. You can see it if you view the whole file.